pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers=[
 [tool.poetry.dependencies]
 python = "^3.10"
 langchain = "^0.1"
-elasticsearch = "^8"
+elasticsearch = ">6"
```

Comment:
Are you sure that Elasticsearch 7 is compatible with semantic search as the 8 ?

Reply:
given we dot not index vectors "as vectors: by default I would allow using it as cache also for the older es clusters